src/RiakClient/Models/MapReduce/RiakMapReduceResultPhase.cs

```diff
@@ -28,10 +28,10 @@ namespace RiakClient.Models.MapReduce
     public class RiakMapReduceResultPhase
     {
         private readonly bool success;
-        private readonly uint phase;
+        private readonly long phase;
```

Comment:
If I switch it back, `RiakMapReduceResultPhase` is no longer CLS-compliant though. Using a `long` doesn't lose data as using an `int` could, so I'll leave it as a `long`.